docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -1221,17 +1213,158 @@ paths:
             $ref: '#/definitions/Problem'
         '409':
           description: |
-            Conflict. There are no empty slots for this subscriptions. The amount of consumers for this subscription
+            Conflict. There are several possible reasond for receiving this status code:
+
+            - There are no empty slots for this subscriptions. The amount of consumers for this subscription
+            already equals the maximal value - the total amount of this subscription partitions.
+
+            - Resetting of subscription cursors request is still in the progress.
+
+            - This endpoint was called for a subscription with balance_mode set to 'custom'.
```

Comment:
Are you going to keep `custom` mode?

Reply:
no sorry, just forgot to remove it. I fixed it now